From: Christoph Hellwig <hch@lst.de>
Date: Mon, 4 May 2020 19:15:19 +0200
Subject: x86/platform/uv: Unexport sn_coherency_id
Patch-mainline: v5.8-rc1
Git-commit: 23e1a65f3c7eb7f3c3f6f6c26b89cdf3bcb128f6
References: jsc#SLE-16407

sn_coherency_id is only used by x2apic_uv_x.c, and uv_sysfs.c, both
of which can't be modular.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Not-acked-by:  Dimitri Sivanich <sivanich@hpe.com>
Cc: Russ Anderson <rja@hpe.com>
Link: https://lkml.kernel.org/r/20200504171527.2845224-4-hch@lst.de
Acked-by: Lee, Chun-Yi <jlee@suse.com>
---
 arch/x86/platform/uv/bios_uv.c |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/x86/platform/uv/bios_uv.c
+++ b/arch/x86/platform/uv/bios_uv.c
@@ -80,7 +80,6 @@ static s64 uv_bios_call_irqsave(enum uv_
 long sn_partition_id;
 EXPORT_SYMBOL_GPL(sn_partition_id);
 long sn_coherency_id;
-EXPORT_SYMBOL_GPL(sn_coherency_id);
 long sn_region_size;
 EXPORT_SYMBOL_GPL(sn_region_size);
 long system_serial_number;
